docs/sdk/web3-react/guides/01-connect-wallet.md

```diff
@@ -1,4 +1,165 @@
+
+Finally, having created the `Web3ContextProvider` component, we can navigate to our [index file](https://github.com/Uniswap/examples/blob/feat/web3-react/web3-react/src/index.tsx) and wrap the whole application with it:
+
+```typescript reference title="Wrapping our app with the web3 context" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/7ac3853bc465aecc428a32be584bbeb833b0a63c/web3-react/src/index.tsx#L16-L22
+```
+
+## Building an Injected Connector
+
+The only parameter that we provided to the `Web3ReactProvider` component is a list of prioritized connectors, declared as `PRIORITIZED_CONNECTORS`. The prioritization ordering is with regards to which connector we want to be active when more than one connector is connected to our application. The list is defined inside our connectors module: 
+
+```typescript reference title="Creating the prioritized Connectors list" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/8c0e36ca8d2ba4718af944094191f39da62a9c5c/web3-react/src/libs/connections.ts#L33-L39
+```
+
+Each one of those connectors lives within its own file, and they all follow a very similar setup pattern. 
+
+An example of a connector in the list is the `InjectedConnector`, which supports wallets that inject an *Ethereum Provider* into the browser window. The most popular example of an injected connector is the *MetaMask* browser extension. To set it up, we import the `initializeConnector` function from the web3-react [core]((https://www.npmjs.com/package/@web3-react/core)) package and the `MetaMask` type from the web3-react [Metamask]((https://www.npmjs.com/package/@web3-react/core)) package:
+
+```typescript reference title="Importing Connector dependencies" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/856dbb002e7f38120554ef226f4309c96ce6ea79/web3-react/src/libs/injected.ts#L1-L2
+```
+
+We then utilize the templated `initializeConnector` function with a type of `MetaMask`:
+
+```typescript reference title="Initializing the MetaMask connector" referenceLinkText="View on Github" customStyling
+https://github.com/Uniswap/examples/blob/81ec93e97b0afded621e177fe5f34fc9f98f80b0/web3-react/src/libs/injected.ts#L7-L9
+```
+
+By passing in `MetaMask` as the templated parameter, we define the function's required input parameters. In this case, the only parameter we need to pass is an instance of `Metamask`, which receives the `actions` and `onError` parameters. The initial parameter defines the actions that `web3-react` can execute on a local store it creates for the connector, while the latter parameter defines the connector's behavior when an error in encountered.
```

Comment:
I think you should only refer to parameters ordinally: "first", "second", etc. Using "initial", "former", "latter" can be confusing.

Reply:
"`web3-react` executes on its local store for the connector (this usually can be passed through without modification)"
"second parameter is a callback invoked when an error occurs"